co-authors-plus.php

```diff
@@ -1446,6 +1434,26 @@ public function update_author_term( $coauthor ) {
 		return $this->get_author_term( $coauthor );
 	}
 
+	/**
+	 * Update author term when user profile is updated.
+	 *
+	 * @param int $userid
+	 */
+	function action_user_profile_update( $userid ) {
+		global $coauthors_plus;
+
+		if( ! current_user_can( 'manage_options' ) ) {
```

Comment:
Is manage_options the permission used for managing authors? It seems this would mean that when a profile page is changed this might not run. Is this intentional?

Reply:
Spot on! You are right. I intended that to be a security measure, so that I would be _doing_it_right, but ended up backfiring. There is the `edit_users` cap for managing users, but it would still not run when one updates their own profile. I'll just remove that check.